src/pages/_user_story.jsx

```diff
@@ -75,6 +74,19 @@ UserStoryPage.propTypes = {
   data: PropTypes.shape({
     userStory: PropTypes.object.isRequired,
   }),
+  pageContext: PropTypes.shape({
+    previous: PropTypes.shape({
+      title: PropTypes.string.isRequired,
+      slug: PropTypes.string.isRequired,
+    }),
+    next: PropTypes.shape({
+      title: PropTypes.string.isRequired,
+      slug: PropTypes.string.isRequired,
+    }),
+  }),
+  data: PropTypes.shape({
+    userStory: PropTypes.object.isRequired,
+  }),
```

Comment:
I don't see these in the UserStory.jsx file before this PR, where is the idea for adding these coming from?